### PR TITLE
Filter icons (#633)

### PR DESCRIPTION
**Pull request by blms**
## What this PR does

- Per #650:
  - Replaces filters icon with new SVG icon, which changes colors based on whether or not filters are active
- Per #524:
  - Replaces filter checkmarks with Phosphor check-circle and check-circle-fill

### CODE REVIEW

File: sitemedia/js/controllers/search_controller.js [hunk 1]
```diff
@@ -36,6 +43,24 @@ export default class extends Controller {
...
+        if (this.filterTargets.some((target) => target.checked)) {
+            this.filterLabelTargets.forEach((label) =>
+                label.classList.add("active")
+            );
+        } else {
+            this.filterLabelTargets.forEach((label) =>
+                label.classList.remove("active")
+            );
+        }
+    }
```
Comment (rlskoeser) [inline sitemedia/js/controllers/search_controller.js:62]: can't this be done server side? we just need to set an `active` class on the filter toggle if there are filters applied, right? how about a simple boolean method on the form that checks if there are filters active 

my inclination is that the we shouldn't display the active status until the filters have been applied to the search; once we implement the reactive search they will be applied automatically, but I think the server side active status still works in that case

**Review by rlskoeser — COMMENTED** [review]
I love how this works! Just a question about whether we can't set filter status to active in the template instead of needing JS

File: sitemedia/js/controllers/search_controller.js (hunk 1 above)
Comment (blms) [reply sitemedia/js/controllers/search_controller.js:62]: Good point!

**Comment by blms** [comment]
Quick question for re-review: does it make sense to return False if there are form errors? I figure it does, but can adjust otherwise

**Review by rlskoeser — APPROVED** [review]
It seems logical to me to return false if the form isn't valid!

Revision looks good to me.

**Pull request merged**